### PR TITLE
docs(): add Utilities and Config sections

### PR DESCRIPTION
**Pull request by liamdebeasi**
This PR adds basic "Config" documentation. It also creates a "Utilities" section for documenting other APIs like this.

### CODE REVIEW

File: src/pages/utilities/config.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      rippleEffect: false,
+      mode: 'md'
+    }), 
+    AppRoutingModule
+  ],
+  ...
+})
+```
+
+In this example, we are disabling the material design ripple effect across the app as well as forcing the app mode to be material design.
```
Comment (brandyscarney) [inline src/pages/utilities/config.md]: Probably say:

> In the above example, we are disabling the Material Design ripple effect across the app, as well as forcing the mode to be Material Design.

File: src/pages/utilities/config.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+| `actionSheetEnter`       | `AnimationBuilder` | Provides a custom enter animation for all `ion-action-sheet`, overriding the default "animation".
+| `actionSheetLeave`       | `AnimationBuilder` | Provides a custom leave animation for all `ion-action-sheet`, overriding the default "animation".
+| `alertEnter`             | `AnimationBuilder` | Provides a custom enter animation for all `ion-alert`, overriding the default "animation".
+| `alertLeave`             | `AnimationBuilder` | Provides a custom leave animation for all `ion-alert`, overriding the default "animation".
+| `animated`               | `boolean`          | When it's set to `false`, disables all animation and transition across the app.
+| `backButtonIcon`         | `string`           | Overrides the default icon in all `<ion-back-button>` components.
+| `backButtonText`         | `string`           | Overrides the default text in all `<ion-back-button>` components.
+| `hardwareBackButton`     | `boolean`          | Wherever ionic will respond to hardware go back buttons in an Android device.
```
Comment (brandyscarney) [inline src/pages/utilities/config.md]: This description doesn't make sense. I think the booleans should be set up in `if` statements like:

```
If `true`, Ionic will respond to the hardware back button in an Android device
```

File: src/pages/utilities/config.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+nextText: ''
+nextUrl: ''
+---
+
+# Config
+
+The Config lets you configure your entire app. You can set the tab button layout, icon mode, animations, and more here.
+
+## Usage
+
```
Comment (brandyscarney) [inline src/pages/utilities/config.md]: Maybe here we should do something similar to components where there is a Basic Usage like:

```
### Basic example

(add some usage here about changing string or boolean types)

### Customizing Animations

(add some usage here on changing an overlay animation)
```

**Comment by liamdebeasi** [comment]
@brandyscarney made the changes you requested. can you take another look?

**Pull request merged**